test/plugins/plugins-mode-support.test.js

```diff
@@ -55,11 +55,9 @@ test('server rendering of a lit element with hydration support', async (t) => {
     development: true,
   });
   await app.importElement(join(tmp, 'element.js'));
-  const result = app.hydrate(
```

Comment:
Are we losing the test for `hydrate` mode, or is it covered by the general `serverRender`?